src/applications/vaos/components/State.jsx

```diff
@@ -70,9 +71,13 @@ export default function State({ state }) {
   }
 
   return (
-    <span role="group">
+    <span>
```

Comment:
Maybe the outer span can be eliminated as well.

Reply:
Good point, let me try that

Reply:
Removed.